MaterialDesignThemes.Wpf/Snackbar.cs

```diff
@@ -1,12 +1,40 @@
 ﻿using System;
+using System.Globalization;
 using System.Windows;
 using System.Windows.Controls;
+using System.Windows.Data;
 using System.Windows.Markup;
 using System.Windows.Media.Animation;
 using System.Windows.Threading;
 
 namespace MaterialDesignThemes.Wpf
 {
+    public enum SnackbarActionButtonPlacementMode
+    {
+        Auto,
+        Inline,
+        SeparateLine
+    }
+
+    public class SnackbarActionButtonPlacementModeConverter : IMultiValueConverter
```

Comment:
Let's make this converter internal and moved into the Converters namespace/folder.

Reply:
sure, I will do that